src/components/notification-banner/success/code.njk

```diff
@@ -5,7 +5,14 @@ layout: layout-example.njk
 
 {% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
 
+{% set html %}
+  <p class="govuk-notification-banner__heading">
+    Training outcome recorded and trainee withdrawn
+  </p>
+  <p>Contact <a class="govuk-notification-banner__link" href="#">becomingateacher@digital.education.gov.uk</a> if you think there's a problem.</p>
```

Comment:
Could we use a fake email address please? this is a live one used by our service.

Reply:
Hi Ed, apologies for that mistake, I'll make the change and thank you for flagging :).